cirq-core/cirq/contrib/quimb/state_vector.py

```diff
@@ -170,8 +170,14 @@ def tensor_expectation_value(
         )
     else:
         tn.rank_simplify(inplace=True)
-    path_info = tn.contract(get='path-info')
-    ram_gb = path_info.largest_intermediate * 128 / 8 / 1024 / 1024 / 1024
+    # Skip path-info evaluation when TensorNetwork consists of scalar Tensors.
+    # Avoid bug in quimb-1.8.0.
+    # Ref: https://github.com/jcmgray/quimb/issues/231
+    if tn.ind_map:
+        path_info = tn.contract(get='path-info')
+        ram_gb = path_info.largest_intermediate * 128 / 8 / 1024 / 1024 / 1024
+    else:
+        ram_gb = 0
```

Comment:
should we raise an error or log a warning?

Reply:
I don't think it is needed.  The error happens for a trivial `TensorNetwork([Tensor()])` for which the memory requirements will be tiny.
The bug was BTW already fixed upstream, but I think we should still merge this so we support the current quimb-1.8.

Reply:
maybe add a TODO to remove this when we upgrade quimb ?

Reply:
Sure, done.